src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -754,6 +754,21 @@ func (r *Cluster) KafkaTLSListeners() []ListenerWithName {
 	return res
 }
 
+// IsKafkaMutualTLSEnabled returns true if Kafka is enabled with mutual TLS
+func (r *Cluster) IsKafkaMutualTLSEnabled() bool {
+	return r.KafkaMutualTLSListener() != nil
+}
+
+// KafkaMutualTLSListener returns the first Kafka mTLS listener
+func (r *Cluster) KafkaMutualTLSListener() *ListenerWithName {
+	for _, el := range r.KafkaTLSListeners() {
```

Comment:
so if this listener is external, console will use external listener? not sure that's desired 🤔 I think it should always use the internal one

Reply:
Updated to always use internal listener :+1: